exegol/utils/WebUtils.py

```diff
@@ -124,7 +125,8 @@ def __runRequest(cls, url: str, service_name: str, headers: Optional[Dict] = Non
         for i in range(retry_count):
             try:
                 try:
-                    response = requests.request(method=method, url=url, timeout=(5, 10), verify=ParametersManager().verify, headers=headers, data=data)
+                    proxy = os.environ.get('HTTP_PROXY') or os.environ.get('HTTPS_PROXY')
```

Comment:
Thank you for the PR!

Both settings should be used: env `HTTP_PROXY` should be set to `http` dict config and `HTTPS_PROXY` to `https`.
Also, it could be great to handle `http_proxy` and `https_proxy` to avoid any case-sensitive error.

Reply:
Thank you for your feedback ! I just modified the code to separate the 2 proxies and support the different formats.